src/pip/_internal/operations/freeze.py

```diff
@@ -164,27 +164,33 @@ def __init__(self, name, req, editable, comments=()):
         self.comments = comments
 
     @classmethod
-    def _init_args_from_dist(cls, dist, dependency_links):
+    def get_requirement_info(cls, dist, dependency_links):
```

Comment:
I don't think this needs to be a `classmethod` since it doesn't actually use cls.